app/views/layout.html

```diff
@@ -18,4 +18,5 @@
 
 {% block body_end %}
   {% include "includes/scripts.html" %}
+  <!-- GOV.UK Prototype kit {{releaseVersion}} -->
```

Comment:
maybe a good use for `<meta name="generator"`

http://stackoverflow.com/questions/3632166/should-i-include-a-meta-generator-tag

`<meta name="generator" content="GOV.UK Prototype kit v2.1.0" />`